src/foundry/client-esm/dice/terms/operator.d.mts

```diff
@@ -0,0 +1,53 @@
+import type RollTerm from "./term.d.mts";
+
+/**
+ * A type of RollTerm used to denote and perform an arithmetic operation.
+ */
+declare class OperatorTerm extends RollTerm {
+  constructor({ operator, options }?: Partial<OperatorTerm.TermData>);
+
+  /** The term's operator value. */
+  operator: OperatorTerm.TermData["operator"];
+
+  /** An object of operators with their precedence values. */
+  static PRECEDENCE: {
+    "+": 10;
+    "-": 10;
+    "*": 20;
+    "/": 20;
+    "%": 20;
+  };
+
+  /**
+   * An array of operators which represent arithmetic operations.
+   * @defaultValue `["+", "-", "*", "/", "%"]`
+   */
+  static OPERATORS: string[]; // TODO: Rely on PRECEDENCE
+
+  static override REGEXP: RegExp;
+
+  /**
+   * An array of additional attributes which should be retained when the term is serialized.
+   * @defaultValue `["operator"]`
+   */
+  static override SERIALIZE_ATTRIBUTES: string[];
+
+  /**
+   * Optional flavor text which modifies and describes this term.
+   * @remarks Operator terms cannot have flavor text
+   */
+  override get flavor(): "";
```

Comment:
Probably leave this more generic for similar reasons to what I said before; any user who _requires_ this to be `""` and not change across Foundry versions is doing something weird. Relying on values being this specific is usually a mistake.